server/api/public.py

```diff
@@ -151,3 +152,22 @@ def compute_batch_comparison_sample_size(
         + 2
     )
     return min(sample_size, contest.ballots)
+
+@api.route(
+    "/file-upload",
+    methods=["POST"],
+)
+@allow_public_access
```

Comment:
Yeah I was starting to worry about this as well and then realized that since this endpoint fails when the config env var is set for s3 we should be ok. Although I'm not sure if there is a similar attack one could do against s3 with the presigned urls if they got them 🤔 

But I will add a any logged in user decorator anyways